site/packages/aa-core/smart-account-client/actions/buildUserOperation.md

```diff
@@ -74,16 +74,16 @@ const batchedUoStruct = await smartAccountClient.buildUserOperation({
 
 // signUserOperation signs the above unsigned user operation struct built
 // using the account connected to the smart account client
-const request = await smartAccountClient.signUserOperation({ uoStruct });
+const request = await smartAccountClient.signUserOperation({ batchedUoStruct });
```

Comment:
I think this was correct as it was

Reply:
I think this line was to refer `batchUoStruct` usage.
the `uoStruct` is refer to the single uo above.
different context.

well that's what i understand when reading the docs, kind a not make sense before, so i made it make sense.

Reply:
ahhh ok then it should be `signUserOperation({ uoStruct: batchedUoStruct })`

Reply:
yeah, you are right, seems I missed to write the param, i'll update soon.

Reply:
hey @nsnull0 any updates here to the PR? are you unblocked or do you want us to review?

Reply:
hey @nsnull0 any updates here to the PR? are you unblocked or do you want us to review?

Reply:
Hi, there will be no further update on me. feel free to review it, if you think not necessary then please close it.
Currently its not blocking my work because I can patch my own work around locally.

Reply:
and yes please, could you help to review it when you have time? thank you.

Reply:
@nsnull0 just made another review. could you please make the requested changes? thanks